client/src/app/components/sell-buy-stock/sell-buy-actions.component.html

```diff
@@ -50,20 +50,20 @@
       >Only Positive Number</span
     >
     <div *ngIf="isMarketOrder" class="sell-buy-field">
-      Estimated Total:<span>-------</span>
+      Estimated Total:<span>{{getTotal() | currency:'USD'}}</span>
     </div>
     <div *ngIf="!isMarketOrder" class="sell-buy-field">
-      Total:<span>-------</span>
+      Total:<span>{{getTotal() | currency:'USD' }}</span>
     </div>
-    <div class="sell-buy-field">Remaining Balance:<span>$ 156.23</span></div>
+    <div class="sell-buy-field">Remaining Balance:<span>{{getRemainingBalance() | currency:'USD'}}</span></div>
   </div>
   <div mat-dialog-actions>
     <button
       class="sell-buy-button"
       mat-flat-button
       color="primary"
       (click)="processActions()"
-      [disabled]="quantity <= 0 || price < 0"
+      [disabled]="quantity <= 0 || price < 0 || getRemainingBalance() < 0"
```

Comment:
because there is quite a bit of logic here, we can refactor this into a function, and write a test for it